src/prefect/server/database/migrations/versions/sqlite/2024_01_22_120214_265eb1a2da4c_create_deployment_schedule_and_add_.py

```diff
@@ -0,0 +1,113 @@
+"""Create `deployment_schedule` and add `Deployment.paused`
```

Comment:
in `work_queue` and `work_pool` this column is called `is_paused` so i'm wondering if we should match for consistency

Reply:
I've already added it to nebula as `paused`. I don't think it's worth going back and making it match, unless you feel strongly.

Reply:
ah okay, `paused` sounds good then